samples/subsys/usb/dfu-next/README.rst

```diff
@@ -0,0 +1,137 @@
+.. zephyr:code-sample:: dfu-next
+   :name: USB DFU
+   :relevant-api: usbd_api
```

Comment:
nit -- doesn't really matter since the usbd_dfu group is not showing in the docs (...yet? I wouldn't mind something like a `/connectivity/usb/device_next/api/dfu_device.html page being added. If nothing else, this makes it less likely that folks browsing the docs at a high level just make the wrong assumption that "oh they probably don't support DFU since I don't see it [here](https://docs.zephyrproject.org/latest/connectivity/usb/device_next/api/index.html))
```suggestion
   :relevant-api: usbd_api usbd_dfu
```

Reply:
Thank you. The new API should be included in the documentation, I really thought I did it.